test/e2e/rte/rte_e2e_suite_test.go

```diff
@@ -35,6 +35,7 @@ import (
 
 	_ "github.com/k8stopologyawareschedwg/resource-topology-exporter/test/e2e/rte"
 	_ "github.com/k8stopologyawareschedwg/resource-topology-exporter/test/e2e/topology_updater"
+	_ "github.com/openshift-kni/numaresources-operator/test/e2e/rte/local"
```

Comment:
Uhm, but if we want to isolate the local tests in their own suite (good), why are we importing it here again?

Reply:
Because I thought there's nothing bad at running the tests again as part of the general suite

Reply:
Ok, I see no problem with that